app/controllers/concerns/authentication.rb

```diff
@@ -16,17 +16,6 @@ module Authentication
   #  - X-Person-Name (first name)
   #  - X-Person-Formal-Name (full name)
 
-  REMOTE_USER_HEADER = 'X-Remote-User'
-  GROUPS_HEADER = 'X-Groups'
-  FIRST_NAME_HEADER = 'X-Person-Name'
-  NAME_HEADER = 'X-Person-Formal-Name'
-
-  # The development environment does not have Shibboleth authentication.
-  # Instead, it will use this hardcoded user.
-  DEV_REMOTE_USER = 'a.user@stanford.edu'
-  DEV_FIRST_NAME = 'A.'
-  DEV_NAME = 'A. User'
```

Comment:
Moved to config.

Reply:
Maybe an outlier, but we did find we needed a remote user header change early in H2 development. I found the `Authentication` module a bit tough to deal with, with that wall of constants screaming at me at the top of the file.